python/ray/node.py

```diff
@@ -243,6 +243,14 @@ def __init__(
         self.metrics_agent_port = self._get_cached_port(
             "metrics_agent_port", default_port=ray_params.metrics_agent_port
         )
+
+        minimal = not ray._private.utils.check_dashboard_dependencies_installed()
+        if ray_params.metrics_export_port and minimal:
+            logger.info(
+                "WARNING: `metrics_export_port` cannot be configured because your "
+                "installation of ray - minimal does not support it. To enable, "
```

Comment:
"ray -minimal" isn't an end user concept. We should probably just tell people what dependencies are missing (or maybe even error since they're explicitly asking for a metrics?).

Reply:
I've just changed the wording to
```
"WARNING: `metrics_export_port` cannot be configured because your "
"installation of ray is missing required dependencies. "
"To export metrics, try installing ray with `pip install ray[default]` "
```

Reply:
I'll leave the determination of whether to error or warn up to someone who has a better idea of the best behaviour. 

I think both are acceptable.

I think the most important thing is that the user will be able to identify the issue. 

If the user is using autoscaler, for instance, I'm not even sure if they will be able to obtain the warning or reason for erroring if we decide to kill their node by erroring.